src/Dofs/DofHandler.jl

```diff
@@ -288,6 +288,23 @@ function cellcoords!(global_coords::Vector{Vec{dim,T}}, grid::Grid{dim,C}, i::In
     end
     return global_coords
 end
+
+function cellnodes!(global_nodes::Vector{Int}, grid::AbstractGrid{dim}, i::Int) where {dim}
+    C = getcelltype(grid)
+    @assert length(global_nodes) == nnodes(C)
+    for j in 1:nnodes(C)
+        global_nodes[j] = getcells(grid, i).nodes[j] ##TODO getnodes(::AbstractCell); only vertices(::AbstractCell) available
+    end 
+    return global_nodes
+end
+
+function cellcoords!(global_coords::Vector{Vec{dim,T}}, grid::AbstractGrid{dim}, i::Int) where {dim,T}
+    C = getcelltype(grid)
+    @assert length(global_coords) == nnodes(C)
+    global_coords .= getcoordinates(grid, i)
+    return global_coords
+end
+
```

Comment:
Not needed after the change in https://github.com/Ferrite-FEM/Ferrite.jl/pull/425/files#diff-8db5ed06971ce1004169862f3e499f55666dd630868843be8a53d30ea385e505?

Reply:
what we really need is: https://github.com/Ferrite-FEM/Ferrite.jl/blob/81440fabd5434820e6b3fb6b7a4529830d90a111/src/Dofs/DofHandler.jl#L315-L329 from Kim's https://github.com/Ferrite-FEM/Ferrite.jl/pull/445 

It's supposed to handle the information for the `reinit!` which needs the actual coords for any grid. The functionality can be recovered by using the already defined functions on `AbstractGrid`, so `getcoordinates` etc.

Reply:
I can either delete the functions completely and wait for Kim's PR or we take the ones from Kim's PR with the slight improvement suggestion I made over there

Reply:
We almost have this since #425 though (see Fredriks comment above). Why not just make the changes to that one? Allowing `AbstractGrid` in the function signature and using `getcells` + `getnodes` should literally have no impact on using the function with the regular `Grid`.

Reply:
Sorry, I think I misunderstood the comment in the beginning. I changed in the latest commit the functions to work for any `<: AbstractGrid` (I hope you meant this @kimauth, otherwise I'll revert it) I had to use the `getcoordinates` dispatch for the `Node` type as an abstraction in between, otherwise grids always need to use `Node` as their node type. In the test example I only used a `NTuple{dim}`, but we can also just assume that `Node` is always used, if you prefer this.

Reply:
Looks like what I meant. 
I think assuming that a node always has a field `x` for the coordinate is not too much of a stretch - but considering that we have `getcoordinates(node)` it's good to use I guess.